.flake8

```diff
@@ -1,19 +1,59 @@
 [flake8]
-# E203 skipped because it annoys black.
-# no pyproject.toml support https://gitlab.com/pycqa/flake8/-/issues/428
-show-source = true
-extend-ignore =
-  E203,
-  F401,
-max-line-length = 100
+
 builtins = _
+
+# Print the total number of errors:
+count = true
+
+# Don't even try to analyze these:
 extend-exclude =
-  # skip tool cache dirs
+  # No need to traverse egg info dir
+  *.egg-info,
+  # tool cache dirs
   *_cache
-  # skip project env vars
+  # project env vars
   .env,
-  # skip adjacent venv
+  # GitHub configs
+  .github,
+  # Cache files of MyPy
+  .mypy_cache,
+  # Cache files of pytest
+  .pytest_cache,
+  # Temp dir of pytest-testmon
+  .tmontmp,
+  # Occasional virtualenv dir
+  .venv
+  # VS Code
+  .vscode,
+  # Temporary build dir
+  build,
+  # This contains sdists and wheels of ansible-navigator that we don't want to check
+  dist,
+  # Metadata of `pip wheel` cmd is autogenerated
+  pip-wheel-metadata,
+  # adjacent venv
   venv
+
+# IMPORTANT: avoid using ignore option, always use extend-ignore instead
+# Completely and unconditionally ignore the following errors:
+extend-ignore =
+  E203,  # skipped because it annoys black
+  F401,  # skipped for unknown reason
+  WPS300,  # "Found local folder import" -- nothing bad about this
+  WPS306,  # "Found class without a base class: *" -- irrelevant under Python 3
+  WPS422,  # "Found future import: *" -- we need these for multipython
+# https://wemake-python-stylegui.de/en/latest/pages/usage/formatter.html
+#format = wemake
+
+# Let's not overcomplicate the code:
+#max-complexity = 10
```

Comment:
Same as above, commented without explanation

Reply:
The explanation is that I want to keep the config close to others where this is uncommented. It should be uncommented at some point but I didn't enable it because it's a new rule that was disabled previously and hasn't undergone the acceptance discussion.